doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,322 @@
+
+Highlights
+**********
+
+* |NCS| v1.0.0 is aligned with the production release of nRF9160.
+* Added a new AWS FOTA sample demonstrating application firmware update through the use of AWS IoT Jobs with MQTT and HTTP.
+* Added secure services in Secure Partition Manager (SPM) for TrustZone devices (nRF9160).
+* Updated the Download Client library with support for HTTPS and IPv6.
+* Included a new version of the nRF BLE Controller.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.0.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF5x Command Line Tools
+     - v9.8.1
+     - `nRF5x Command Line Tools`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.18 or later.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`aws_fota_sample` - shows how to perform over-the-air firmware updates of an nRF9160 through the use of AWS IoT Jobs with MQTT and HTTP.
+  * :ref:`secure_services` - demonstrates using the reboot and random number services.
+
+* Added the following libraries:
+
+  * :ref:`lib_fota_download` - handles Firmware Over The Air (FOTA) downloads.
+  * :ref:`lib_aws_fota` - combines the :ref:`lib_aws_jobs` and :ref:`lib_fota_download` libraries to create a user-friendly library that can perform firmware-over-the-air (FOTA) update using HTTP and MQTT TLS.
+  * :ref:`at_cmd_readme` - facilitates handling of AT Commands by multiple modules.
+  * :ref:`lib_aws_jobs` - facilitates communication with the AWS IoT Jobs service.
```

Comment:
Since this is referenced by `lib_aws_fota` should it be moved before that entry?

Reply:
makes sense, moving